units/src/fee_rate/mod.rs

```diff
@@ -117,34 +120,34 @@ impl FeeRate {
     #[must_use]
     pub const fn checked_sub(self, rhs: u64) -> Option<Self> {
         // No `map()` in const context.
-        match self.to_sat_per_kwu().checked_sub(rhs) {
+        match self.to_sat_per_kwu_floor().checked_sub(rhs) {
             Some(res) => Some(Self::from_sat_per_kwu(res)),
             None => None,
         }
     }
 }
 
 impl From<FeeRate> for u64 {
```

Comment:
In 4831689f0181c458dc6acf118414f56fcfe56bd1:

Can we just delete this impl? Not to channel Kix too hard but it's really unclear what it should mean.

Reply:
Ah, done in 95755218353f84b71fb3457ab8f1a9fff796e72d